internal/providers/terraform/google/kms_crypto_key_test.go

```diff
@@ -0,0 +1,46 @@
+package google_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/testutil"
+)
+
+func TestKMSCryptoKey(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	tf := `
+				resource "google_kms_crypto_key" "my_keys" {
```

Comment:
the indentation looks odd here, we can treat these TF code snippets like other code when indenting